engine/src/build/config/apple/apple_sdk.gni

```diff
@@ -0,0 +1,17 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
```

Comment:
I think we're still following the convention that the year under the engine directory is 2013.

Reply:
Oh interesting. I didn't even notice it, but I copy-pasted this from the mac_sdk.py/ios_sdk.py etc. Which still have the old Chromium headers!

Reply:
Done.